src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -113,7 +113,7 @@ void fillInnerParts() {
         expenseListPanel = new ExpenseListPanel(logic.getFilteredExpenseList());
         expenseListPanelPlaceholder.getChildren().add(expenseListPanel.getRoot());
 
-        resultDisplay = new ResultDisplay();
+        resultDisplay = new ResultDisplay("Budget allowance for the month: $" + logic.getBudget().getBudget());
```

Comment:
I think is okay since budget is now displayed as a result (result of SetBudgetCommand, new budget reflected after Add/Delete commands)...? Unless if we looking to have a different UI where budget display is separate from normal expenses results display, then we should have a different name....? I think this can be discussed during meeting!